example/transactions/Basic_38_TxTypeCancel.js

```diff
@@ -0,0 +1,63 @@
+const ethers = require("ethers");
+const { KlaytnWallet } = require("../../dist/src/ethers"); // require("@klaytn/sdk-ethers");
+
+const fs = require('fs')
+const sender_priv = fs.readFileSync('./example/privateKey', 'utf8') 
+
+const sender = '0x3208ca99480f82bfe240ca6bc06110cd12bb6366' 
+const reciever = '0xc40b6909eb7085590e1c26cb3becc25368e249e9'  
+
+//
+// TxTypeCancel
+// https://docs.klaytn.foundation/content/klaytn/design/transactions/basic#txtypecancel
+// 
+//   type: Must be 0x38,
+//
+// 1) send ValueTransfer tx with the original nonce+1  
+// 2) send Cancel tx with the original nonce+1 
+// 3) send ValueTransfer tx with the original nonce 
+//    then you can see Cancel tx with the original nonce+1 
+// 
+async function main() {
+  const provider = new ethers.providers.JsonRpcProvider('https://public-en-baobab.klaytn.net')
+  const wallet = new KlaytnWallet(sender_priv, provider);
+   
```

Comment:
You don't have to use populateTransaction() to find the nonce.
See https://docs.ethers.org/v5/api/signer/#Signer-getTransactionCount.

```
nonce = await wallet.getTransactionCount()
```

Reply:
Updated examples with wallet.getTransactionCount() function.

https://github.com/blukat29/klaytn-sdk-design/pull/2/commits/ae78e198f7eefc002160f19f3b1b5737739618a2